CHANGELOG_PENDING.md

```diff
@@ -16,6 +16,9 @@
 - [codegen] Reduce time to execute `pulumi convert` and some YAML programs, depending on providers used, by up to 3 seconds.
   [#10444](https://github.com/pulumi/pulumi/pull/10444)
 
+- [sdk/go] enable direct compilation via go build(set `PULUMI_USE_GO_RUN=true` to opt out)
+  [#10375](https://github.com/pulumi/pulumi/pull/10375)
```

Comment:
I strongly believe the polarity of this should be flipped, and we should use a runtime option or env var to opt in to using `go build` and the new behavior.

As I don't currently see the env var used in this PR, requesting changes.

Reply:
took feedback and forgot to update the CHANGELOG_PENDING. it's now `PULUMI_GO_USE_RUN`. nice catch, will fix.